app/branding.js

```diff
@@ -272,11 +230,7 @@ export function getFeaturedMarkets(quotes = []) {
         ["USD", "RUDEX.WLS"],
         ["USD", "RUDEX.SMOKE"],
         ["CNY", "BTS"],
-        ["CNY", "OPEN.BTC"],
         ["CNY", "USD"],
-        ["CNY", "OPEN.ETH"],
-        ["CNY", "YOYOW"],
-        ["CNY", "OCT"],
         ["CNY", "GDEX.BTC"],
         ["CNY", "GDEX.ETH"],
         ["CNY", "GDEX.EOS"],
```

Comment:
GDEX announced to delist
* HPB
* BTM
* QTUM
* NEO
* GAS

BTW Rudex delisted a few tokens too, I don't remember if they have created a pull request.